lib/sinon/util/core/index.js

```diff
@@ -2,7 +2,6 @@
 
 module.exports = {
     calledInOrder: require("./called-in-order"),
-    configureLogError: require("./log_error"),
```

Comment:
If this was accessible, we should document that it was removed even if it was previously deprecated.